<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="message_field">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="send_button">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTip">
       <string>Send message to VRChat Chatbox</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="log_field">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
